dotCMS/src/main/java/com/dotcms/publisher/util/dependencies/PushPublishigDependencyProcesor.java

```diff
@@ -789,9 +880,46 @@ private void setLanguageVariables() {
             final String langVarsQuery = "+contentType:" + LanguageVariableAPI.LANGUAGEVARIABLE ;
             final List<Contentlet> langVariables = contentletAPI.search(langVarsQuery, 0, -1, StringPool.BLANK, user, false);
 
-            tryToAddAllAndProcessDependencies(PusheableAsset.CONTENTLET, langVariables);
+            tryToAddAllAndProcessDependencies(PusheableAsset.CONTENTLET, langVariables,
+                    ManifestReason.INCLUDE_AUTOMATIC_BY_DOTCMS.getMessage());
         }catch (Exception e){
             Logger.error(this, e.getMessage(),e);
         }
     }
+
+    private static boolean isHostFolderSystem(Object dependency) {
+
+        try {
+            final Host  systemHost = APILocator.getHostAPI().findSystemHost();
+            final Folder systemFolder = APILocator.getFolderAPI().findSystemFolder();
+
+            if (Contentlet.class.isInstance(dependency)){
+                return ((Contentlet) dependency).getIdentifier().equals(systemHost.getIdentifier());
```

Comment:
you can use:

Content.class.cast(dependency).getIdentifier().equals(systemHost.getIdentifier());

Reply:
done https://github.com/dotCMS/core/pull/20632/commits/ea4bb1a64a9d7627f6136822114de495a1887e10